www/docs/en/dev/platform_plugin_versioning_ref/index.md

```diff
@@ -51,43 +52,55 @@ After running the above command, the resulting config.xml looks like :
     ...
 </xml>
 ```
+After running the above command, the resulting package.json looks like :
+
+```bash
+"cordova": {"platforms": ["android",]},"dependencies": {"cordova-android": "~4.0.0"}
+```
+The '--nosave' flag prevents adding and deleting specified platforms from config.xml and package.json. To prevent saving a platform, you issue the following command :
+
+```bash
+$ cordova platform add <platform[@<version>] | directory | git_url> --nosave
+```
 
 Some examples :
 
-  * **'cordova platform add android --save'** => retrieves the pinned version of the android platform, adds it to the project and then updates config.xml.
-  * **'cordova platform add android@3.7.0 --save'** => retrieves the android platform, version 3.7.0 from npm, adds it to the project and then updates config.xml.
-  * **'cordova platform add android@https://github.com/apache/cordova-android.git​ --save'** => clones the specified cordova-android git repository, adds the android platform to the project, then updates config.xml and point its version to the specified git-url.
-  * **'cordova platform add C:/path/to/android/platform --save'** => retrieves the android platform from the specified directory, adds it to the project, then updates config.xml and point to the directory.
+  * **'cordova platform add android'** => retrieves the pinned version of the android platform, adds it to the project and then updates config.xml and package.json.
+  * **'cordova platform add android@3.7.0'** => retrieves the android platform, version 3.7.0 from npm, adds it to the project and then updates config.xml and package.json.
+  * **'cordova platform add android@https://github.com/apache/cordova-android.git​'** => clones the specified cordova-android git repository, adds the android platform to the project, then updates config.xml and package.json and points its version to the specified git-url.
```

Comment:
can we changed this to `cordova platform add https://github.com/apache/cordova-android.git​`. Also, instead of saying clones the specified codova-android git repository, lets say npm installs the specified cordova-android from the git repository.